doc/rados/operations/crush-map-edits.rst

```diff
@@ -1,25 +1,26 @@
-Manually editing a CRUSH Map
-============================
+Manually editing the CRUSH Map
+==============================
 
-.. note:: Manually editing the CRUSH map is an advanced
-	  administrator operation.  All CRUSH changes that are
-	  necessary for the overwhelming majority of installations are
-	  possible via the standard ceph CLI and do not require manual
-	  CRUSH map edits.  If you have identified a use case where
-	  manual edits *are* necessary with recent Ceph releases, consider
-	  contacting the Ceph developers so that future versions of Ceph
-	  can obviate your corner case.
+.. note:: Manually editing the CRUSH map is an advanced administrator
+   operation. For the overwhelming majority of installations, any necessary
```

Comment:
s/any necessary//
ymmv

Reply:
Good eye. Accepted.

I'm also going to cut out "overwhelming".